packages/cozy-intent/src/api/services/NativeService.ts

```diff
@@ -94,9 +92,13 @@ export class NativeService {
     if (this.messengerRegister[uri].connection)
       throw new Error(interpolate(strings.errorInitWebview, { uri }))
 
-    this.messengerRegister[uri].connection = await this.initWebview(
-      this.messengerRegister[uri].messenger
-    )
+    try {
+      this.messengerRegister[uri].connection = await this.initWebview(
+        this.messengerRegister[uri].messenger
+      )
+    } catch {
+      // Handshake failed, what now?
```

Comment:
let's log error here to check if this error occurs a lot

Reply:
Updated